webextension/background.js

```diff
@@ -1,3 +1,47 @@
+function setPopupIcon(theme) {
+  let icons = {
+    16: "img/container-site-d-24.png",
+    32: "img/container-site-d-48.png"
+  };
+  if (theme === "firefox-compact-dark@mozilla.org") {
+    icons = {
+      16: "img/container-site-w-24.png",
+      32: "img/container-site-w-48.png"
+    };
+  }
+  browser.browserAction.setIcon({
+    path: icons
+  });
+}
+
+let existingTheme = null;
```

Comment:
I don't like to have global variables like this. Can you please create an object and move this code into it?
This object can do: 1 checkTheme, updateTheme and so on. Basically all the theme management.